### PR TITLE
Issue/13326 threat detail

### PR DESCRIPTION
**Pull request by malinajirka**
Parent issue #13326

This PR updates ThreatDetail screen
- Displays Fixed header next to the icon for Fixed threats
- Displays Found header below the icon for Fixed threats
- Displays Found header next to the icon for non-fixed threats (current, ignored)
- Update icons background based on threat status (red for current, grey for ignored, green for fixed)

To test:
1. Make sure Scan feature flag is enabled
2. Open threat detail for Current, Fixed and Ignored threats and make sure it matches the behavior described above

Notes:
1. `threat_details_list_header.xml ` is using styles from Jetpack styles (eg. Jetpack.Header). I'm not sure if that's fine, wdyt?
2. Updating the tests was a bit tricky - I think the existing tests are trying to test too much within one method. We might want to consider splitting them into multiple smaller tests where each test verifies only a "single" behavior.  
 
PR submission checklist:

- [x] I have considered adding unit tests where possible.
- [x] I have considered adding accessibility improvements for my changes.
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

File: WordPress/src/test/java/org/wordpress/android/ui/jetpack/scan/details/ThreatDetailsListItemsBuilderTest.kt [hunk 1]
```diff
@@ -17,26 +18,37 @@ import org.wordpress.android.fluxc.model.scan.threat.ThreatModel.Fixable
...
 import org.wordpress.android.ui.jetpack.common.JetpackListItemState.ActionButtonState
 import org.wordpress.android.ui.jetpack.common.JetpackListItemState.DescriptionState
 import org.wordpress.android.ui.jetpack.common.JetpackListItemState.HeaderState
-import org.wordpress.android.ui.jetpack.common.JetpackListItemState.IconState
 import org.wordpress.android.ui.jetpack.scan.TEST_FILE_PATH
 import org.wordpress.android.ui.jetpack.scan.ThreatTestData
 import org.wordpress.android.ui.jetpack.scan.builders.ThreatItemBuilder
 import org.wordpress.android.ui.jetpack.scan.details.ThreatDetailsListItemState.ThreatContextLinesItemState
 import org.wordpress.android.ui.jetpack.scan.details.ThreatDetailsListItemState.ThreatFileNameState
+import org.wordpress.android.ui.jetpack.scan.details.ThreatDetailsListItemState.ThreatDetailHeaderState
```
Comment (ashiagr) [inline WordPress/src/test/java/org/wordpress/android/ui/jetpack/scan/details/ThreatDetailsListItemsBuilderTest.kt]: Minor 🔍: Import-ordering needed here.

**Review by ashiagr — APPROVED** [review]
Thanks @malinajirka! All looks good to me 🌟. I just left a minor comment.

Regarding your notes

> threat_details_list_header.xml is using styles from Jetpack styles (eg. Jetpack.Header). I'm not sure if that's fine, wdyt?

How about we override `Jetpack.Header` in `scan_style`:
`<style name="Scan.ThreatDetails.Header" parent="Jetpack.Header">`
and use this style on the threat details header?

>  I think the existing tests are trying to test too much within one method. We might want to consider splitting them into multiple smaller tests where each test verifies only a "single" behavior.

We can surely do this, let's discuss more on how granular they should be in our Android sync call.

File: WordPress/src/test/java/org/wordpress/android/ui/jetpack/scan/details/ThreatDetailsListItemsBuilderTest.kt (hunk 1 above)
Comment (malinajirka) [reply WordPress/src/test/java/org/wordpress/android/ui/jetpack/scan/details/ThreatDetailsListItemsBuilderTest.kt]: Fixed in https://github.com/wordpress-mobile/WordPress-Android/pull/13893/commits/978361bccc6342e577570a48c3f1aa9977198637

**Comment by malinajirka** [comment]
Thanks for the review @ashiagr! 


> How about we override Jetpack.Header in scan_style: <style name="Scan.ThreatDetails.Header" parent="Jetpack.Header">
and use this style on the threat details header?

Fixed in 4f152ae

**Review by ashiagr — APPROVED** [review]
LGTM, thanks!

**Pull request merged**